### PR TITLE
Added checkfiles for storage and coordinating resource children number

### PR DESCRIPTION
**Pull request by ganning127**
Coordinating resources (with children):
Screenshot 2023-06-21 at 1 27 48 PM

Coordinating resources (no children):
Screenshot 2023-06-21 at 1 26 29 PM

Storage resources (with children):
Screenshot 2023-06-21 at 1 30 18 PM

Storage resources (no children):
Screenshot 2023-06-21 at 1 34 04 PM

### CODE REVIEW

File: src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Link } from '@reach/router'
+
+export default {
+    name: "Coordinating resources have at least one child",
+    description: `Checks if all coordinating resources (deferred, random, passthru, load_balanced, replication, compound)`,
+    minimum_server_version: "4.2.0",
+    maximum_server_version: "",
+    interval_in_seconds: 3600,
```
Comment (trel) [inline src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js:9]: consider daily?   no strong opinions on this one.

**Review by trel — COMMENTED** [review]
looks pretty great - please squash to single commit - no # yet

**Comment by trel** [comment]
you know how to squash....  do you know how to separate existing commits?   a new powerup?

File: src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js (hunk 1 above)
Comment (ganning127) [reply src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js:9]: Yep, daily seems more reasonable to me here

Comment (ganning127) [reply src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js:9]: Should we bump the storage one up to daily as well?

File: src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Link } from '@reach/router'
+
+export default {
+    name: "Coordinating resources have at least one child",
+    description: `Checks if all coordinating resources (deferred, random, passthru, load_balanced, replication, compound)`,
```
Comment (trel) [inline src/data/checkfiles/default_checkfiles/validate-coord-resc-children.js:6]: this isn't quite a complete sentence...

**Comment by trel** [comment]
good, squash carefully back to two (110 into 110)... and still no #.

**Comment by ganning127** [comment]
done!

**Comment by trel** [comment]
let's # em.

**Comment by ganning127** [comment]
added #

**Pull request merged**